shelley/chain-and-ledger/executable-spec/src/EpochBoundary.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE TemplateHaskell       #-}
 {-# LANGUAGE TupleSections         #-}
+{-# OPTIONS_GHC -Wno-name-shadowing #-}
```

Comment:
closing parentheses could be aligned with the above

Reply:
We  are about to add a `stylish-haskell` config that would align that as:
```
{-# LANGUAGE TemplateHaskell #-}
{-# LANGUAGE TupleSections #-}
{-# OPTIONS_GHC -Wno-name-shadowing #-}
```
so I thought it was not worth it.

Reply:
I removed all the `-Wno-name-shadowing` and used `hiding` instead.